internal/helm/controller/reconcile.go

```diff
@@ -316,6 +316,16 @@ func (r HelmOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.
 	}
 
 	log.Info("Reconciled release")
+	reason := types.ReasonUpgradeSuccessful
+	if expectedRelease.Version == 1 {
+		reason = types.ReasonInstallSuccessful
+	}
```

Comment:
Now it makes more sense for me 👍

Reply:
When I ran the `helm ls` command it showed:
```
NAME        	NAMESPACE	REVISION	UPDATED                                	STATUS  	CHART      	APP VERSION
nginx-sample	default  	1       	2020-08-13 14:09:57.443165995 -0400 EDT	deployed	nginx-0.1.0	1.16.0  
``` 

So I thought `Version` is `APP VERSION` which means its not going to be `1` for fresh install case. Then I double checked and it seems like `Version` is infact the `REVISION` from the output above so I changed it back to what Joe suggested earlier.